spring-cloud-stream-reactive/src/test/java/org/springframework/cloud/stream/reactive/StreamListenerReactorTests.java

```diff
@@ -165,7 +334,17 @@ public void receive(@Input(Processor.INPUT) Flux<String> input, @Output(Processo
 
 	@EnableBinding(Processor.class)
 	@EnableAutoConfiguration
-	public static class TestInputOutputArgsWithMessage {
+	public static class TestInputOutputArgs2 {
+
+		@StreamListener(Processor.INPUT)
```

Comment:
I don't believe we should support this - and all the other cases that involve a mix of parameters that are annotated with `@Input`/`@Output` and parameters that are not annotated - either all of them should be annotated, or none (that would really only allow a single parameter in practice). In my opinion, having a mix of parameters is too hard to follow.